src/httpserver/handlers/handler_function_selector.clj

```diff
@@ -0,0 +1,8 @@
+(ns httpserver.handlers.handler_function_selector
```

Comment:
Also, namespaces should use `kebab-case` while filenames use `snake_case` (yes, it's annoying)